client/src/views/Simulation/components/ParametersPanel.vue

```diff
@@ -46,9 +46,7 @@
     </settings-bar>
     <div class="parameters">
       <figure class="parameters-graph" ref="figure"><svg /></figure>
-      <div v-if="noDisplayedParameters" class="alert alert-info m-3">
-        Use the model visualization on the left to add/remove parameters.
-      </div>
+      <message-display v-if="noDisplayedParameters" :message="`Use the model visualization on the left or the ➕ and 🚫 buttons above to add/remove parameters.`" />
```

Comment:
If instead of the `message` prop you used a slot in the `message-display` component, you will then be able to leverage `<font-awesome>` components. and it will make it easier to write and edit.

Reply:
Done.